src/components/widgets-layouts/figure/figure.less

```diff
@@ -0,0 +1,10 @@
+.analytics .layouts.figure {
```

Comment:
No styles are being applied to indicate this is a column or scrollable as specified in the JIRA. 
In screenshot below I have added entries and they just extend the height of the widget. 
<img width="652" alt="screen shot 2018-03-19 at 11 23 56" src="https://user-images.githubusercontent.com/7486451/37592896-0e4292cc-2b68-11e8-8d95-133aa35b3dc9.png">